test/Twig/Tests/LexerTest.php

```diff
@@ -16,7 +16,7 @@ public function testStreamSource()
         $lexer = new Twig_Lexer($env);
 
         $env->disableDebug();
-        $this->assertSame('', $lexer->tokenize('foo')->getSource());
+        $this->assertSame('foo', $lexer->tokenize('foo')->getSource());
```

Comment:
testing the difference between the debug and non-debug mode here is useless now.

this needs to be tested in the place where it makes a difference now.